src/components/Header/Header.jsx

```diff
@@ -0,0 +1,255 @@
+import React, { useState } from "react"
+import { useLocation } from 'react-router-dom'
+import { NavLink } from "react-router-dom"
+import CyrillicToTranslit from 'cyrillic-to-translit-js'
+import ListenerWidth from "./ListenerWidth"
+import {
+    ContainerHeder, LogoHeader, LeftBlock, PointsBalance, NumberBalance,
```

Comment:
If the content doesn't fit the screen, it's more readable to make each word from a new row

Reply:
Thanks for the lesson) I'll fix it. I will try to write like this in the future.